hw/ip_templates/clkmgr/rtl/clkmgr.sv.tpl

```diff
@@ -96,10 +98,12 @@ rg_srcs = list(sorted({sig['src_name'] for sig
   // jittery enable to ast
   output mubi4_t jitter_en_o,
 
+% if len(derived_clks) > 0:
```

Comment:
I suggest you move this input up so it is part of the block you intend to remove, and the code will become more readable.